srcDoc/release_notes.rst

```diff
@@ -98,6 +98,15 @@ Added 2 new Governance profiles to the list of generated governance files in
 
 As well as their respective signed versions (`signed_`...).
 
+Added news print formats for data display (#280)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Now perftest supports different print formats, manage with `-outputFormat <format>`,
+ which can be: csv (default), json or legacy (the previous one).
+
+Another flag has been added, `-noPrintHeaders`, to not print the headers row of
+ intervals and summaries, and only show numbers.
```

Comment:
```
*RTI Perftest* default output format has been improved. This format should now be compatible with the `CSV` format standard, therefore allowing the copy into spreadsheet editors or easier parsing via scripting. This change has been introduced in a way in which the output retains its readability, by correctly aligning rows and columns and by printing headers for each column.

With this change we also introduced the option of changing or customizing the output format by using the `-outputFormat <format>`. At this point the supported values are `csv` (default), `json` or `legacy` (Referring to the previous output used by *RTI Perftest*.

Another flag has been added, `-noPrintHeaders`, in order to skip printing the headers rows (for the summaries and also for the interval information).
 ```

Reply:
I have to improve my English. Changes applied.